python/ray/__init__.py

```diff
@@ -19,6 +20,10 @@
                  "override this by explicitly setting OMP_NUM_THREADS.")
     os.environ["OMP_NUM_THREADS"] = "1"
 
+if sys.platform == "darwin" and not gethostname().endswith(".local"):
+    logger.warning("The GCS Service may not correctly resolve your hostname."
+                   "To fix this please look at issue #7837")
```

Comment:
Can you provide a link to the issue? or rather, maybe a one-line fix?

Reply:
URL instead of "#7837"